app/src/main/java/doit/study/droid/sqlite/helper/DatabaseHelper.java

```diff
@@ -0,0 +1,390 @@
+    private static final String KEY_ANSWER = "answer";
+    private static final String KEY_IS_RIGHT = "is_right";
+
+    // TAGS Table - column names
+    private static final String KEY_TAG_NAME = "tag_name";
+
+    // QUESTION_TAG Table - column names
+    private static final String KEY_QUESTION_ID = "question_id";
+    private static final String KEY_TAG_ID = "tag_id";
+
+    // Table Create Statements
+    // Todo table create statement
+    private static final String CREATE_TABLE_QUESTION = "CREATE TABLE "
+            + TABLE_QUESTION + "("
+            + KEY_ID + " INTEGER PRIMARY KEY,"
+            + KEY_QUESTION + " TEXT,"
+            + KEY_DOC_REF + " TEXT,"
+            + KEY_IS_BINARY + " INTEGER DEFAULT 0,"
+            + KEY_ANSWER_IS_TRUE + " INTEGER DEFAULT 0,"
+            + KEY_CREATED_AT
+            + " DATETIME DEFAULT CURRENT_TIMESTAMP" + ")";
+
+    // Tag table create statement
+    private static final String CREATE_TABLE_TAG = "CREATE TABLE " + TABLE_TAG
+            + "(" + KEY_ID + " INTEGER PRIMARY KEY," + KEY_TAG_NAME + " TEXT,"
+            + KEY_CREATED_AT + " DATETIME DEFAULT CURRENT_TIMESTAMP" + ")";
+
+    // todo_tag table create statement
+    private static final String CREATE_TABLE_QUESTION_TAG = "CREATE TABLE "
+            + TABLE_QUESTION_TAG + "(" + KEY_ID + " INTEGER PRIMARY KEY,"
+            + KEY_QUESTION_ID + " INTEGER," + KEY_TAG_ID + " INTEGER,"
+            + KEY_CREATED_AT + " DATETIME DEFAULT CURRENT_TIMESTAMP" + ")";
+
+    private static final String CREATE_TABLE_ANSWER = "CREATE TABLE "
+            + TABLE_ANSWER + "("
+            + KEY_ID + " INTEGER PRIMARY KEY,"
+            + KEY_ANSWER + " TEXT,"
+            + KEY_IS_RIGHT + " INTEGER DEFAULT 0,"
+            + KEY_QUESTION_ID + " INTEGER,"
+            + KEY_CREATED_AT
+            + " DATETIME DEFAULT CURRENT_TIMESTAMP" + ")";
+
+    private final Integer mTestFile = R.raw.quiz;
+    private Context mContext;
+
+    public DatabaseHelper(Context context) {
+        super(context, DATABASE_NAME, null, DATABASE_VERSION);
+        this.mContext = context;
+    }
+
+    @Override
+    public void onCreate(SQLiteDatabase db) {
+
+        // creating required tables
+        db.execSQL(CREATE_TABLE_QUESTION);
+        db.execSQL(CREATE_TABLE_ANSWER);
+        db.execSQL(CREATE_TABLE_TAG);
+        db.execSQL(CREATE_TABLE_QUESTION_TAG);
+    }
+
+    @Override
+    public void onUpgrade(SQLiteDatabase db, int oldVersion, int newVersion) {
+        // on upgrade drop older tables
+        db.execSQL("DROP TABLE IF EXISTS " + TABLE_QUESTION);
+        db.execSQL("DROP TABLE IF EXISTS " + TABLE_ANSWER);
+        db.execSQL("DROP TABLE IF EXISTS " + TABLE_TAG);
+        db.execSQL("DROP TABLE IF EXISTS " + TABLE_QUESTION_TAG);
+
+        // create new tables
+        onCreate(db);
+    }
+
+    public void importFromFile() {
+
+        File flagFile = new File(mContext.getFilesDir(), "imported");
+        if (flagFile.exists()) return;
```

Comment:
To insert new questions need some counter like DATABASE_VERSION.
We can store it as Shared Preferences.